lib/containers/container_images.pm

```diff
@@ -107,16 +109,19 @@ sub test_opensuse_based_image {
         my $pretty_version = $version =~ s/-SP/ SP/r;
         validate_script_output("$runtime container run --entrypoint '/bin/bash' --rm $image -c 'cat /etc/os-release'", sub { /PRETTY_NAME="SUSE Linux Enterprise Server $pretty_version"/ });
 
-        if (is_sle('=12-SP3', $version)) {
-            my $plugin = '/usr/lib/zypp/plugins/services/container-suseconnect';
-            assert_script_run "$runtime container run --entrypoint '/bin/bash' --rm $image -c '$plugin'";
-        } else {
-            my $plugin = '/usr/lib/zypp/plugins/services/container-suseconnect-zypp';
-            assert_script_run "$runtime container run --entrypoint '/bin/bash' --rm $image -c '$plugin -v'";
-            script_run "$runtime container run --entrypoint '/bin/bash' --rm $image -c '$plugin lp'", 420;
-            script_run "$runtime container run --entrypoint '/bin/bash' --rm $image -c '$plugin lm'", 420;
+        unless (is_sle_container_supported) {
```

Comment:
This should be `if (is_sle_container_supported) { … }`.

Reply:
i see my mistake. i got you

Reply:
fixed. thanks man